src/styles/01-colors/_colors-brand.css

```diff
@@ -1,23 +1,33 @@
-/** Brand Colors */
+/**
+ * Brand Colors
+ *
+ * Color Reference in Figma Main File
+ * https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=15454%3A47007
+ *
+ */
 :root {
-  /** What's On is used for Exhibitions & Events. */
-  --section-whats-on-primary: #8a1a9c;
-  --section-whats-on-secondary: #7c1097;
+  /* Brand fills are the NYPL primary and secondary colors */
+  --brand-primary: #c60917;
+  --brand-secondary: #760000;
+
+  /* What's On is used for Exhibitions & Events. */
+  --section-whats-on-primary: #242424;
+  --section-whats-on-secondary: #000000;
 
```

Comment:
One thing we should think about is whether we want to try giving the hex values some color name (example "fucshia") which describes the color rather than something like "--section-whats-on-primary".  This would be helpful when designing components based off of Figma designs that use certain colors but don't necessarily belong to, for example, the "What's On" section.  This approach would be similar to what you are already doing for `--brand-primary ` and `--brand secondary`.  For example:  ```--section-whats-on-primary: var(--cl-fucshia);```

Reply:
@kaplanmichael 
Ideally, the names of the CSS vars match the color names used in the Figma file.  Which I believe they do, at this point.  I am hoping to start talking more seriously about global NYPL design tokens soon and this discussion will definitley be part of that.